testapi.pm

```diff
@@ -714,6 +714,19 @@ sub wait_still_screen {
     return 0;
 }
 
+=head2 assert_still_screen
+
+  assert_still_screen([$args...])
+
+Run C<wait_still_screen> but C<die> if screen changed within timeout. Look
+into C<wait_still_screen> for details.
+
+=cut
+
+sub assert_still_screen(@) {
+    wait_still_screen(@_) or die 'assert_still_screen failed to detect a still screen';
```

Comment:
In other cases where we assert things we just record a result and set the overall test result to failed. Exceptions are usually thrown when there's a problem with the test code or backend itself and I assume they lead to an "incomplete" result. So have you tested this in a "fullstack-way" to see whether your change really produces the right *overall* behavior?

Reply:
That's the same approach as for assert_screen_change. Any exception in test scope is handled properly to forward as test failure, but not when just calling the function in test directly